### ISSUE TITLE
the token of a list is empty

### ISSUE THREAD

**Issue by kongou-ae** [redpen-cc/redpen#638]
Does redpen tokenize the sentence of a list ? I look that the token is empty.

# sample doc

```
ああああああああああ。

* ご飯を食べますか。それともパンですか
    * aaaaa
* aaaaa
```

# test code
```javascript
function validateSection(section){
    console = {
        log:print
    };

    for (i = 0; i < section.getListBlocks().length; i++){
        for (j = 0; j < section.getListBlock(i).getListElements().length; j++){
            console.log(j)
            console.log(section.getListBlock(i).getListElement(j).getSentences())
        }

    }
}
```

# result 
```
[Sentence{links=[], lineNumber=3, content='ご飯を食べますか。', startPositionOffset=2,isFirstSentence=false, tokens=[], offsetMap=[LineOffset(3, 2), LineOffset(3, 3), LineOffset(3, 4), LineOffset(3, 5), LineOffset(3, 6), LineOffset(3, 7), LineOffset(3, 8), LineOffset(3, 9), LineOffset(3, 10)]}, Sentence{links=[], lineNumber=3, content='それともパンですか', startPositionOffset=11, isFirstSentence=false, tokens=[], offsetMap=[LineOffset(3, 11), LineOffset(3, 12), LineOffset(3, 13), LineOffset(3, 14), LineOffset(3, 15), LineOffset(3, 16), LineOffset(3, 17), LineOffset(3, 18), LineOffset(3, 19)]}]
1
[Sentence{links=[], lineNumber=4, content='aaaaa', startPositionOffset=6, isFirstSentence=false, tokens=[], offsetMap=[LineOffset(4, 6), LineOffset(4, 7), LineOffset(4, 8), LineOffset(4, 9), LineOffset(4, 10)]}]
2
[Sentence{links=[], lineNumber=5, content='aaaaa', startPositionOffset=2, isFirstSentence=false, tokens=[], offsetMap=[LineOffset(5, 2), LineOffset(5, 3), LineOffset(5, 4), LineOffset(5, 5), LineOffset(5, 6)]}]
```

**Comment by takahi-i** [comment]
Hello @kongou-ae. I made the fix with #640. Hopefully this patch works for you.

**Comment by kongou-ae** [comment]
Thanks!!

```
[Sentence{links=[], lineNumber=3, content='ご飯を食べますか。', startPositionOffset=2,isFirstSentence=false, tokens=[TokenElement{surface='ご飯', offset=0, tags=[名詞, 一般, *, *, *, *, ご飯, ゴハン, ゴハン]}, TokenElement{surface='を', offset=2, tags=[助詞, 格助詞, 一般, *, *, *, を, ヲ, ヲ]}, TokenElement{surface='食べ', offset=3, tags=[動詞, 自立, *, *, 一段, 連用形, 食べる, タベ, タベ]}, TokenElement{surface='ます', offset=5, tags=[助動詞, *, *, *, 特殊・マス, 基本形, ます, マス, マス]}, TokenElement{surface='か', offset=7, tags=[助詞, 副助詞／並立助詞／終助詞, *, *, *, *, か, カ, カ]}, TokenElement{surface='。', offset=8, tags=[記号, 句点, *, *, *, *, 。, 。, 。]}], offsetMap=[LineO
ffset(3, 2), LineOffset(3, 3), LineOffset(3, 4), LineOffset(3, 5), LineOffset(3, 6), LineOffset(3, 7), LineOffset(3, 8), LineOffset(3, 9), LineOffset(3, 10)]}, Sentence{links=[], lineNumber=3, content='それともパンですか', startPositionOffset=11, isFirstSentence=false, tokens=[TokenElement{surface='それとも', offset=0, tags=[接続詞, *, *, *, *, *, それとも, ソレトモ, ソレトモ]}, TokenElement{surface='パン', offset=4, tags=[名詞, 一般, *, *, *, *, パン, パン, パン]}, TokenElement{surface='です', offset=6, tags=[助動詞, *, *, *, 特殊・デス, 基本形, です, デス, デス]}, TokenElement{surface='か', offset=8, tags=[助詞, 副助詞／並立助詞／終助詞, *, *, *, *, か, カ, カ]}], offsetMap=[LineOffset(3, 11), LineOffset(3, 12), LineOffset(3, 13), LineOffset(3, 14), LineOffset(3, 15), LineOffset(3, 16
), LineOffset(3, 17), LineOffset(3, 18), LineOffset(3, 19)]}]
```
